### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by developit**
Hi! I came across your preact variant on Twitter and when I installed it and got the keys setup, a few things didn't seem to be functioning quite correctly.  Not sure if it's useful, but I've split up the three issues I came across into three fix commits here. Cheers!

### CODE REVIEW

**Comment by i-like-robots** [comment]
Hey @developit, thanks for your input!

I got a little too trigger happy pushing this up in the first instance. I've since switched the behaviour to be a little closer to the original - I [pushed that up](https://github.com/i-like-robots/preact-tube-tracker/commit/529c970c034b839814fbcfd602193ca64f9db4a6) before I saw your PR but I'm very appreciative that you took time to help me out.

File: src/common/utils.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export function formatQueryString (props) {
   const queryString = []
 
   for (const prop in props) {
-    if (props.hasOwnProperty(prop)) {
+    if (props.hasOwnProperty(prop) && props[prop]!=null) {
```
Comment (i-like-robots) [inline src/common/utils.js:44]: Good call

**Comment by i-like-robots** [comment]
I've made sure everything is working now as it should (376d2804cd72b90478552579cc55965812c658b0). Thank you again for taking the time to find issues and reporting them. My apologies too for being so hasty in pushing this out there!

**Comment by developit** [comment]
No problem at all, it was fun to check out.

**Pull request closed without merging**